webapp/src/main/resources/application.sql.yml

```diff
@@ -67,3 +68,13 @@ sql:
 
     sqlFindByCreator: >-
       SELECT * FROM upload_student_group_batch WHERE creator=:creator
+
+  import:
+    createQuery: >-
+      INSERT INTO import (status, content, contentType, digest, batch) VALUES
+        (0, 5, 'admin group modification', :digest, :batch)
```

Comment:
I am of the same opinion. But I have to admit that due to the lack of a better field and time pressure I am doing the same with the legacy data load.